diff --git a/dist/config.js b/dist/config.js
index 2d0c9f07f0e17c510469f51350d8bd044403eeef..959e3f8329eb7bc37bf8bd377d67d3988950c8d8 100644
--- a/dist/config.js
+++ b/dist/config.js
@@ -109,7 +109,7 @@ async function loadExtConfigJS(rootDir, extConfigName, extConfigFilePath) {
             extConfigType: 'js',
             extConfigName,
             extConfigFilePath: extConfigFilePath,
-            extConfig: require(extConfigFilePath),
+            extConfig: await import(extConfigFilePath),
         };
     }
     catch (e) {
