diff --git a/dist/esm/index.js b/dist/esm/index.js
index 2034a1a1a6f08dbaa6f8f86879a9c127271ec23b..581aeb537114289b17c68bc3eabb4e7f21be9f6e 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -1,6 +1,6 @@
 import { registerPlugin } from '@capacitor/core';
 const SplashScreen = registerPlugin('SplashScreen', {
-    web: () => import('./web').then(m => new m.SplashScreenWeb()),
+    web: () => import(/* webpackChunkName: 'capacitor' */ './web').then(m => new m.SplashScreenWeb()),
 });
 export * from './definitions';
 export { SplashScreen };
