diff --git a/dist/config.js b/dist/config.js
index 2d0c9f07f0e17c510469f51350d8bd044403eeef..56e8400df754f33f47fdfe23d96961ea3cbd68cc 100644
--- a/dist/config.js
+++ b/dist/config.js
@@ -85,7 +85,7 @@ async function loadExtConfigTS(rootDir, extConfigName, extConfigFilePath) {
             (0, errors_1.fatal)('Could not find installation of TypeScript.\n' +
                 `To use ${colors_1.default.strong(extConfigName)} files, you must install TypeScript in your project, e.g. w/ ${colors_1.default.input('npm install -D typescript')}`);
         }
-        const ts = require(tsPath); // eslint-disable-line @typescript-eslint/no-var-requires
+        const ts = (await import(tsPath)).default; // eslint-disable-line @typescript-eslint/no-var-requires
         const extConfigObject = (0, node_1.requireTS)(ts, extConfigFilePath);
         const extConfig = (_a = extConfigObject.default) !== null && _a !== void 0 ? _a : extConfigObject;
         return {
@@ -109,7 +109,7 @@ async function loadExtConfigJS(rootDir, extConfigName, extConfigFilePath) {
             extConfigType: 'js',
             extConfigName,
             extConfigFilePath: extConfigFilePath,
-            extConfig: require(extConfigFilePath),
+            extConfig: (await import(extConfigFilePath)).default,
         };
     }
     catch (e) {
