diff --git a/dist/declarations/src/types/state.d.ts b/dist/declarations/src/types/state.d.ts
index 15af087941b9f33b7d4f73e49aca9278c205d8ec..4a308b269bc1c84edd8dbac4c8b3248ef3bb2b5b 100644
--- a/dist/declarations/src/types/state.d.ts
+++ b/dist/declarations/src/types/state.d.ts
@@ -175,7 +175,7 @@ export type CommonGestureState = {
      * The arguments passed to the bind function (only relevant in React when
      * using `<div {...bind(someArgument)} />`)
      */
-    args?: any;
+    args?: any[];
 };
 export type CoordinatesState = CommonGestureState & {
     /**
