diff --git a/dist/whyDidYouRender.js b/dist/whyDidYouRender.js
index 6b8284bbcfcebdeadf8bf24e596312684714901d..01ff1b7c58186f7b67696685a4719827defd82a3 100644
--- a/dist/whyDidYouRender.js
+++ b/dist/whyDidYouRender.js
@@ -429,6 +429,26 @@
       }
       return trackDiff(new Set(a), new Set(b), diffsAccumulator, pathString, diffTypes.deepEquals);
     }
+    if (lodash.isMap(a) && lodash.isMap(b)) {
+      if (a.size !== b.size) {
+        return trackDiff(new Map(a), new Map(b), diffsAccumulator, pathString, diffTypes.different);
+      }
+      var _iterator = _createForOfIteratorHelper(a),
+        _step;
+      try {
+        for (_iterator.s(); !(_step = _iterator.n()).done;) {
+          var valA = _step.value[0];
+          if (!b.has(valA)) {
+            return trackDiff(new Set(a), new Set(b), diffsAccumulator, pathString, diffTypes.different);
+          }
+        }
+      } catch (err) {
+        _iterator.e(err);
+      } finally {
+        _iterator.f();
+      }
+      return trackDiff(new Map(a), new Map(b), diffsAccumulator, pathString, diffTypes.deepEquals);
+    }
     if (lodash.isDate(a) && lodash.isDate(b)) {
       return a.getTime() === b.getTime() ? trackDiff(new Date(a), new Date(b), diffsAccumulator, pathString, diffTypes.date) : trackDiff(new Date(a), new Date(b), diffsAccumulator, pathString, diffTypes.different);
     }
