diff --git a/index.d.ts b/index.d.ts
index 075093a74e04d43ea186f079750bc0ad8f886d0a..9d31147cd7d3f562bd11aeacc5d81673818f3973 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -6,7 +6,7 @@ type BaseOptions = {
 
 	@default Infinity
 	*/
-	readonly concurrency?: number;
+	readonly concurrency: number;
 };
 
 export type Options = BaseOptions & {
@@ -98,7 +98,7 @@ console.log(result);
 export default function pMap<Element, NewElement>(
 	input: AsyncIterable<Element | Promise<Element>> | Iterable<Element | Promise<Element>>,
 	mapper: Mapper<Element, NewElement>,
-	options?: Options
+	options: Options
 ): Promise<Array<Exclude<NewElement, typeof pMapSkip>>>;
 
 /**
