diff --git a/node_modules/@capacitor/app/dist/esm/index.js b/node_modules/@capacitor/app/dist/esm/index.js
index 6834c6d..6473631 100644
--- a/node_modules/@capacitor/app/dist/esm/index.js
+++ b/node_modules/@capacitor/app/dist/esm/index.js
@@ -1,6 +1,6 @@
 import { registerPlugin } from '@capacitor/core';
 const App = registerPlugin('App', {
-    web: () => import('./web').then(m => new m.AppWeb()),
+    web: () => import(/* webpackChunkName: 'capacitor' */ './web').then(m => new m.AppWeb()),
 });
 export * from './definitions';
 export { App };
