diff --git a/node_modules/@types/lodash/index.d.ts b/node_modules/@types/lodash/index.d.ts
index 8d1ec9f..4c8c110 100755
--- a/node_modules/@types/lodash/index.d.ts
+++ b/node_modules/@types/lodash/index.d.ts
@@ -35,11 +35,11 @@ declare namespace _ {
 // Backward compatibility with --target es5
 declare global {
     // tslint:disable-next-line:no-empty-interface
-    interface Set<T> { }
+    // interface Set<T> { }
     // tslint:disable-next-line:no-empty-interface
-    interface Map<K, V> { }
+    // interface Map<K, V> { }
     // tslint:disable-next-line:no-empty-interface
-    interface WeakSet<T> { }
+    // interface WeakSet<T> { }
     // tslint:disable-next-line:no-empty-interface
-    interface WeakMap<K extends object, V> { }
+    // interface WeakMap<K extends object, V> { }
 }
