diff --git a/node_modules/constate/CHANGELOG.md b/node_modules/constate/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/constate/LICENSE b/node_modules/constate/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/constate/README.md b/node_modules/constate/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/constate/dist/constate.cjs.js b/node_modules/constate/dist/constate.cjs.js
old mode 100644
new mode 100755
diff --git a/node_modules/constate/dist/constate.es.js b/node_modules/constate/dist/constate.es.js
old mode 100644
new mode 100755
diff --git a/node_modules/constate/dist/constate.min.js b/node_modules/constate/dist/constate.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/constate/dist/ts/src/index.d.ts b/node_modules/constate/dist/ts/src/index.d.ts
old mode 100644
new mode 100755
index 12243b8..844726f
--- a/node_modules/constate/dist/ts/src/index.d.ts
+++ b/node_modules/constate/dist/ts/src/index.d.ts
@@ -1,5 +1,5 @@
 import * as React from "react";
-declare type Selector<Value> = (value: Value) => any;
+declare type Selector<Value> = (value: Value) => Memoed<any>;
 declare type SelectorHooks<Selectors> = {
     [K in keyof Selectors]: () => Selectors[K] extends (...args: any) => infer R ? R : never;
 };
@@ -8,5 +8,5 @@ declare type ConstateTuple<Props, Value, Selectors extends Selector<Value>[]> =
     React.FC<Props>,
     ...Hooks<Value, Selectors>
 ];
-declare function constate<Props, Value, Selectors extends Selector<Value>[]>(useValue: (props: Props) => Value, ...selectors: Selectors): ConstateTuple<Props, Value, Selectors>;
+declare function constate<Props, Value, Selectors extends Selector<Value>[]>(useValue: (props: Props) => Memoed<Value>, ...selectors: Selectors): ConstateTuple<Props, Value, Selectors>;
 export default constate;
diff --git a/node_modules/constate/dist/ts/test/index.test.d.ts b/node_modules/constate/dist/ts/test/index.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/constate/src/index.tsx b/node_modules/constate/src/index.tsx
old mode 100644
new mode 100755
