diff --git a/node_modules/knex/lib/query/querycompiler.js b/node_modules/knex/lib/query/querycompiler.js
index a4a09d2..69fe13d 100644
--- a/node_modules/knex/lib/query/querycompiler.js
+++ b/node_modules/knex/lib/query/querycompiler.js
@@ -424,9 +424,12 @@ class QueryCompiler {
           this.bindingsHolder
         );
       } else {
+        const isLateral = join.joinType.endsWith('lateral');
+        const joinType = isLateral ? join.joinType.slice(0, -7) : join.joinType;
         sql +=
-          join.joinType +
+          joinType +
           ' join ' +
+          (isLateral ? 'lateral ' : '') +
           wrap_(
             table,
             undefined,
