diff --git a/node_modules/knex/lib/formatter/wrappingFormatter.js b/node_modules/knex/lib/formatter/wrappingFormatter.js
index c9fe7ec..6725ddc 100644
--- a/node_modules/knex/lib/formatter/wrappingFormatter.js
+++ b/node_modules/knex/lib/formatter/wrappingFormatter.js
@@ -51,6 +51,8 @@ const operators = transform(
     ['?', '\\?'],
     ['?|', '\\?|'],
     ['?&', '\\?&'],
+    'is distinct from',
+    'is not distinct from',
   ],
   (result, key) => {
     if (Array.isArray(key)) {
diff --git a/node_modules/knex/lib/query/querybuilder.js b/node_modules/knex/lib/query/querybuilder.js
index 096e35e..c758f84 100644
--- a/node_modules/knex/lib/query/querybuilder.js
+++ b/node_modules/knex/lib/query/querybuilder.js
@@ -732,7 +732,7 @@ class Builder extends EventEmitter {
   }
 
   // Adds a `order by` clause to the query.
-  orderBy(column, direction, nulls = '') {
+  orderBy(column, direction, nulls) {
     if (Array.isArray(column)) {
       return this._orderByArray(column);
     }
diff --git a/node_modules/knex/lib/query/querycompiler.js b/node_modules/knex/lib/query/querycompiler.js
index a4a09d2..69fe13d 100644
--- a/node_modules/knex/lib/query/querycompiler.js
+++ b/node_modules/knex/lib/query/querycompiler.js
@@ -424,9 +424,12 @@ class QueryCompiler {
           this.bindingsHolder
         );
       } else {
+        const isLateral = join.joinType.endsWith('lateral');
+        const joinType = isLateral ? join.joinType.slice(0, -7) : join.joinType;
         sql +=
-          join.joinType +
+          joinType +
           ' join ' +
+          (isLateral ? 'lateral ' : '') +
           wrap_(
             table,
             undefined,
