diff --git a/node_modules/typescript/lib/lib.dom.d.ts b/node_modules/typescript/lib/lib.dom.d.ts
index ac946ba..c1414b6 100644
--- a/node_modules/typescript/lib/lib.dom.d.ts
+++ b/node_modules/typescript/lib/lib.dom.d.ts
@@ -22622,7 +22622,7 @@ interface URLSearchParams {

 declare var URLSearchParams: {
     prototype: URLSearchParams;
-    new(init?: string[][] | Record<string, string> | string | URLSearchParams): URLSearchParams;
+    new(init?: string[][] | Partial<Record<string, string | number>> | string | URLSearchParams): URLSearchParams;
 };

 /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/UserActivation) */
diff --git a/node_modules/typescript/lib/lib.webworker.d.ts b/node_modules/typescript/lib/lib.webworker.d.ts
index 82f0b0b..34e23c3 100644
--- a/node_modules/typescript/lib/lib.webworker.d.ts
+++ b/node_modules/typescript/lib/lib.webworker.d.ts
@@ -5650,7 +5650,7 @@ interface URLSearchParams {

 declare var URLSearchParams: {
     prototype: URLSearchParams;
-    new(init?: string[][] | Record<string, string> | string | URLSearchParams): URLSearchParams;
+    new(init?: string[][] | Partial<Record<string, string | number>> | string | URLSearchParams): URLSearchParams;
 };
